wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1327,184 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+			<!--============AudioClipManagement===================-->
+			<xs:complexType name="AudioClipConfiguration">
+				<xs:sequence>					
+					<xs:element name="Enabled" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation> Enables/disables the audio clip.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Name" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Name of the audio clip.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Token" type="tt:ReferenceToken">
+					<xs:annotation>
+						<xs:documentation> Unique Token associated with the audio clip.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="AudioOutputToken" type="tt:ReferenceToken">
+					<xs:annotation>
+						<xs:documentation> Token of the physical Audio Output.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Type" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Format of the audio clip. For definitions see tt:SupportedAudioClipFormat parameter.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="RepeatCycles" type="xs:int">
+					<xs:annotation>
+						<xs:documentation> Number of times the audio clip can be repeated when it is played. The value 0 means no repeat i.e. play the audio clip, does not repeat after the first play. 1 means play the audio clip and repeat once after the first play. -1 means repeat the audio clip infinite after the first play until the clip is stopped by the user operation.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Gain" type="xs:int" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional Gain to control volume of the audio clip when it is played.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="ScheduleToken" type="tt:ReferenceToken" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional Schedule Token associated with the audio clip configuration. Audio clip can be played when configured schedule is triggered.</xs:documentation>
```

Comment:
"Audio clip can be played when configured schedule is triggered"
what does this mean, is it that it is only allowed to be played during active scedhule or that it should be played when schedule becomes active?

Reply:
remove schedule support?

Reply:
Audio clip can be played when schedule becomes active.

Reply:
Can you describe the use-case for this?

Reply:
Schedule Token is used to play the audio clip when associated schedule is triggered. Example use case is playing audio clip during shop opening at morning 6 am , lunch break 1pm  , closing time 9 pm in a day.